Rakefile

```diff
@@ -231,7 +307,8 @@ def create_directory_bundle(target, prefix = nil)
   sh "cp -pR %s %s"                  % [vendor_dir, lib_dir]
   sh "cp %s Gemfile Gemfile.lock %s" % [spec_path, dest_vendor_dir]
 
-  GEMSPEC.require_paths.each do |path|
+
+  GEMSPEC.require_paths.select { |path| path !~ /.rbenv/ }.each do |path|
```

Comment:
@netshade Can you explain why this filter exists? Bundling into `vendor/bundle` isn't a thing that is possible with this, afaict.